libpod/container_internal_linux.go

```diff
@@ -1786,6 +1795,31 @@ func (c *Container) getHosts() string {
 			}
 		}
 	}
+
+	// Add gateway entry
+	var depCtr *Container
+	if c.config.NetNsCtr != "" {
+		// ignoring the error because there isn't anything to do
+		depCtr, _ = c.getRootNetNsDepCtr()
+	} else if len(c.state.NetworkStatus) != 0 {
+		depCtr = c
+	} else {
+		depCtr = nil
+	}
+
+	if depCtr != nil {
+		for _, pluginResultsRaw := range depCtr.state.NetworkStatus {
+			pluginResult, _ := cnitypes.GetResult(pluginResultsRaw)
+			for _, ip := range pluginResult.IPs {
+				hosts += fmt.Sprintf("%s host.gateway.internal\n", ip.Gateway)
+			}
+		}
+	} else if c.config.NetMode.IsSlirp4netns() {
+		hosts += fmt.Sprintf("%s host.gateway.internal\n", slirp4netnsGateway)
+	} else {
+		logrus.Debug("unexpected network configuration while determining host.gateway.internal address")
```

Comment:
Is this unexpected, or would this just be the situation if I run a container with `podman run --net=none ...`

IE Is this worth logging?

Reply:
This would be the case for `podman run --net=none` so I suppose it isn't that unexpected. Should I just quietly continue or change up the debug message?

Reply:
@rhatdan bump just in case you didn't see my response